readme.md

```diff
@@ -211,7 +211,7 @@ The use cases that can currently be addressed are below. See [Tests](stripe-serv
 ```
   mvn clean install
   
-  # Skip gpg signing 
+  //Skip gpg signing
```

Comment:
That looks like an unrelated change.

Reply:
it is. I snuck it in because my markdown plugin exposed it as an error.